app/src/assets/Custom/RenderCategory.js

```
@@ -0,0 +1,59 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useIsFocused } from '@react-navigation/native';
+import React, { useEffect, useState } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+const RenderCategory = ({ item, isSelected, onPress }) => {
+  const name = language === "fr" && item.french_name ? item.french_name : item.name;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)